numpy/core/__init__.py

```diff
@@ -53,6 +53,17 @@
 del os
 
 from . import umath
+
+# Test that multiarray is a pure python module wrapping _multiarray_umath,
+# and not the old c-extension module.
+if not (getattr(multiarray, '_multiarray_umath', None) and 
+        getattr(umath, '_multiarray_umath', None)):
+    msg = ("Something is wrong with the NumPy installation. "
+       "While importing we detected an older version of "
+       "numpy. One method of fixing this is to repeatedly uninstall "
```

Comment:
Can we show `np.__path__` to user in some way here, to help them work out where they're other installations are?

Reply:
fixed